### PR TITLE
Update sim launcher gui docs

### PR DESCRIPTION
**Pull request by antonelepfl**

### CODE REVIEW

**Comment by antonelepfl** [comment]
@alex4200 please have a look and if it's ok then Luca can merge it.

File: source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst [hunk 1]
```diff
@@ -20,9 +20,6 @@ From the **top** you can configure:
 -  ``Model``: Circuit target to be loaded by the simulator.
```
Comment (alex4200) [inline source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst]: Should be named 'population'.

File: source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst [hunk 2]
```diff
@@ -48,6 +45,7 @@ From the **middle** you can configure:
       -  *Duration*: Time length of stimulus duration, given in
          milliseconds(ms)
       -  *Number of synapses*: The number of synapses to create
+      -  *Pattern*: Type of stimulus to inject
       -  *Lambda*: Configure the random distribution
       -  *Weight*: The strength of the created synapses
```
Comment (alex4200) [inline source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst]: Below: 'Target' should be renamed 'population'.

File: source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst [hunk 3]
```diff
@@ -20,9 +20,6 @@ From the **top** you can configure:
 -  ``Model``: Circuit target to be loaded by the simulator.
 -  ``Duration``: Time length of stimulus duration, given in
    milliseconds(ms)
--  ``Forward Skip``: Time length in milliseconds (ms) of running the
-   simulation without stimulus and without reporting. This is to get the
-   neuronal cells past any initial transience
 -  ``View Simulations``: From here you can see all the history of
    simulations (explained in section **2**)
```
Comment (alex4200) [inline source/online_usecases/brain_area_circuit_in_silico_experiments/hippocampus/configure_run_brainregion_preconf_model_data/configure_run_brainregion_preconf_model_data.rst]: Below: 'Target' should be renamed 'population'.

**Comment by antonelepfl** [comment]
I have resolved the comments.
@lbologna I think we can merge this.

**Comment by lbologna** [comment]
Ok. Merged.

**Pull request merged**